src/write/op.rs

```diff
@@ -1100,417 +1100,417 @@ mod tests {
-                                read::Operation::Address { address: 23 },
-                            ),
-                            (
-                                &|x| x.op_const_type(entry_id, vec![23].into()),
-                                Operation::ConstantType(entry_id, vec![23].into()),
-                                read::Operation::TypedLiteral {
-                                    base_type: entry_offset,
-                                    value: read::EndianSlice::new(&[23], LittleEndian),
-                                },
-                            ),
-                            (
-                                &|x| x.op_convert(None),
-                                Operation::Convert(None),
-                                read::Operation::Convert {
-                                    base_type: read::UnitOffset(0),
-                                },
-                            ),
-                            (
-                                &|x| x.op_convert(Some(entry_id)),
-                                Operation::Convert(Some(entry_id)),
-                                read::Operation::Convert {
-                                    base_type: entry_offset,
-                                },
-                            ),
-                            (
-                                &|x| x.op_reinterpret(None),
-                                Operation::Reinterpret(None),
-                                read::Operation::Reinterpret {
-                                    base_type: read::UnitOffset(0),
-                                },
-                            ),
-                            (
-                                &|x| x.op_reinterpret(Some(entry_id)),
-                                Operation::Reinterpret(Some(entry_id)),
-                                read::Operation::Reinterpret {
-                                    base_type: entry_offset,
-                                },
-                            ),
-                            (
-                                &|x| x.op_wasm_local(1000),
-                                Operation::WasmLocal(1000),
-                                read::Operation::WasmLocal { index: 1000 },
-                            ),
-                            (
-                                &|x| x.op_wasm_global(1000),
-                                Operation::WasmGlobal(1000),
-                                read::Operation::WasmGlobal { index: 1000 },
-                            ),
-                            (
-                                &|x| x.op_wasm_stack(1000),
-                                Operation::WasmStack(1000),
-                                read::Operation::WasmStack { index: 1000 },
-                            ),
-                        ];
+                    #[allow(clippy::type_complexity)]
+                    let operations: &[(&dyn Fn(&mut Expression), _, _)] = &[
```

Comment:
I assume this change was an attempt to fix `clippy::type_complexity`? I prefer not to do it.